cmd/zfs_object_agent/zoa/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "zoa"
+version = "0.1.0"
+authors = ["Manoj Joseph <manoj.joseph@delphix.com>"]
```

Would you mind using the corporate "authors" as we have in the other Cargo.toml files?
```suggestion
authors = ["Delphix"]
```

---

Fixed